com.unity.uiwidgets/Runtime/engine2/UIWidgetsPanel.cs

```diff
@@ -76,16 +85,7 @@ public partial class UIWidgetsPanel : RawImage, IUIWidgetsWindow {
             }
         }
 
-        public static bool ShowDebugLog {
-            get { return _ShowDebugLog; }
-            set {
-                foreach (var panel in panels) {
-                    panel.m_ShowDebugLog = value;
-                }
-
-                _ShowDebugLog = value;
-            }
-        }
+        public static bool ShowDebugLog { get; set; }
```

Comment:
we don't need this anymore

this reason why we need it is that: in our previous design, all UIWidgetsPanels share the same showDebugLog flag. Now since we have changed this design, we don't need it anymore